/mnt/hwfile/huqinghao/miniconda3/bin/python
2024-05-15 23:16:44,967 - INFO - Note: detected 128 virtual cores but NumExpr set to maximum of 64, check "NUMEXPR_MAX_THREADS" environment variable.
2024-05-15 23:16:44,967 - INFO - Note: NumExpr detected 128 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.
2024-05-15 23:16:44,968 - INFO - NumExpr defaulting to 8 threads.
2024-05-15 23:16:45,892	INFO scripts.py:1182 -- Did not find any active Ray processes.
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638cac0>: Failed to establish a new connection: [Errno 111] Connection refused'))

==========
== CUDA ==
==========

CUDA Version 12.1.0

Container image Copyright (c) 2016-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.

This container image and its contents are governed by the NVIDIA Deep Learning Container License.
By pulling and using the container, you accept the terms and conditions of this license:
https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license

A copy of this license is made available in this container at /NGC-DL-CONTAINER-LICENSE for your convenience.

*************************
** DEPRECATION NOTICE! **
*************************
THIS IMAGE IS DEPRECATED and is scheduled for DELETION.
    https://gitlab.com/nvidia/container-images/cuda/blob/master/doc/support-policy.md

Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d450>: Failed to establish a new connection: [Errno 111] Connection refused'))
INFO 05-15 23:17:02 api_server.py:191] vLLM API server version 0.3.3
INFO 05-15 23:17:02 api_server.py:192] args: Namespace(host=None, port=8000, uvicorn_log_level='info', allow_credentials=False, allowed_origins=['*'], allowed_methods=['*'], allowed_headers=['*'], api_key=None, served_model_name=None, lora_modules=[], delta_modules=[Delta(name='delta-1', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-1'), Delta(name='delta-2', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-2'), Delta(name='delta-3', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-3'), Delta(name='delta-4', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-4'), Delta(name='delta-5', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-5'), Delta(name='delta-6', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-6'), Delta(name='delta-7', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-7'), Delta(name='delta-8', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-8'), Delta(name='delta-9', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-9'), Delta(name='delta-10', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-10'), Delta(name='delta-11', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-11'), Delta(name='delta-12', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-12'), Delta(name='delta-13', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-13'), Delta(name='delta-14', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-14'), Delta(name='delta-15', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-15'), Delta(name='delta-16', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-16')], swap_modules=[], chat_template=None, response_role='assistant', ssl_keyfile=None, ssl_certfile=None, ssl_ca_certs=None, ssl_cert_reqs=0, root_path=None, middleware=[], model='/vllm/.idea/full_models/Llama-2-13b-hf', tokenizer=None, revision=None, code_revision=None, tokenizer_revision=None, tokenizer_mode='auto', trust_remote_code=False, download_dir=None, load_format='auto', dtype='auto', kv_cache_dtype='auto', max_model_len=None, worker_use_ray=False, pipeline_parallel_size=1, tensor_parallel_size=4, max_parallel_loading_workers=None, ray_workers_use_nsight=False, block_size=16, enable_prefix_caching=False, seed=0, swap_space=4, gpu_memory_utilization=0.85, max_num_batched_tokens=None, max_num_seqs=256, max_logprobs=5, disable_log_stats=False, quantization=None, enforce_eager=True, max_context_len_to_capture=8192, disable_custom_all_reduce=False, tokenizer_pool_size=0, tokenizer_pool_type='ray', tokenizer_pool_extra_config=None, enable_lora=False, max_loras=1, max_lora_rank=64, lora_extra_vocab_size=256, lora_dtype='auto', max_cpu_loras=32, enable_delta=True, max_deltas=16, max_cpu_deltas=32, max_delta_bitwidth=4, device='auto', image_input_type=None, image_token_id=None, image_input_shape=None, image_feature_size=None, scheduler_delay_factor=0.0, enable_prefetch=False, scheduler_policy='fcfs', max_swap_slots=1, max_cpu_models=4, enable_swap=False, engine_use_ray=False, disable_log_requests=True, max_log_len=None)
2024-05-15 23:17:05,692	INFO worker.py:1749 -- Started a local Ray instance.
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638c970>: Failed to establish a new connection: [Errno 111] Connection refused'))
INFO 05-15 23:17:08 llm_engine.py:90] Initializing an LLM engine (v0.3.3) with config: model='/vllm/.idea/full_models/Llama-2-13b-hf', tokenizer='/vllm/.idea/full_models/Llama-2-13b-hf', tokenizer_mode=auto, revision=None, tokenizer_revision=None, trust_remote_code=False, dtype=torch.float16, lora_config=None, delta_config=DeltaConfig(max_deltas=16, max_bitwidth=4, delta_dtype=torch.int8, max_cpu_deltas=32, delta_extra_vocab_size=0, pack_factor=Fraction(2, 1), kernel=<QuantKernel.TRITON: 'triton'>), swap_config=None, max_seq_len=4096, download_dir=None, load_format=auto, tensor_parallel_size=4, disable_custom_all_reduce=False, quantization=None, enforce_eager=True, kv_cache_dtype=auto, device_config=cuda, seed=0)
INFO 05-15 23:17:08 llm_engine.py:243] ignored verifying delta_config
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638dd20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e5f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638eec0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f3d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e7a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ded0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d600>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638cdc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f790>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d000>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d6f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638dfc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e890>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ee30>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e9b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e0e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d810>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d5d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ca60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d0f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638dba0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e470>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ed40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f190>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e680>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f7f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f220>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e320>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638da50>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d4e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d420>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638db40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e410>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ece0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f940>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638fdf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ceb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f9d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e830>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638df60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d690>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8735f980a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e980>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f2e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f400>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f7c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8736367b20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d300>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f0a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ee00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e530>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638dc60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e2c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e200>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ead0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e8f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f730>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8735f98520>: Failed to establish a new connection: [Errno 111] Connection refused'))
Using bitblas
WARNING 05-15 23:26:47 models.py:48] Using bitblas delta modules
INFO 05-15 23:26:47 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=84425)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f37279cdc60>
[36m(RayWorkerVllm pid=84425)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=84425)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=84425)[0m     self.remove()
[36m(RayWorkerVllm pid=84425)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=84425)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=84425)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=84761)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f54f49e5c60>
[36m(RayWorkerVllm pid=84761)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=84761)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=84761)[0m     self.remove()
[36m(RayWorkerVllm pid=84761)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=84761)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=84761)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638cf70>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=84761)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f54f49e5c60>
[36m(RayWorkerVllm pid=84761)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=84761)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=84761)[0m     self.remove()
[36m(RayWorkerVllm pid=84761)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=84761)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=84761)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f610>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f070>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e920>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e7d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8735f989a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ed70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f9a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638fd90>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d330>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8735f98340>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638cf70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d360>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638fc70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638eb30>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8736367910>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8735f98790>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ed70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638fb50>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d510>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638cf40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8735f98bb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ceb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f6d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f220>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e350>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e020>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8735f992a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e650>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638da20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638eb00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638fb20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8735f98820>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ff70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f7c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f400>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f2e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e230>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d990>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e290>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ec20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638fa30>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638fc40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638c220>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638ca00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638faf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638fa00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e890>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638da50>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8735f99720>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638e500>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638edd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638f490>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638fd90>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=84425)[0m Using bitblas
[36m(RayWorkerVllm pid=84609)[0m Using bitblas
[36m(RayWorkerVllm pid=84761)[0m Using bitblas
[36m(RayWorkerVllm pid=84425)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=84425)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=84761)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=84761)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=84761)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=84761)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:35:25 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:35:25 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:35:30 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:35:30 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:35:35 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:35:35 selector.py:16] Using FlashAttention backend.
INFO 05-15 23:35:49 model_runner.py:161] Loading model weights took 6.1114 GB
INFO 05-15 23:35:49 worker_manager.py:297] Starting prefetching thread
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:35:50 model_runner.py:161] Loading model weights took 6.1114 GB
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:35:50 worker_manager.py:297] Starting prefetching thread
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:35:50 model_runner.py:161] Loading model weights took 6.1114 GB
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:35:50 worker_manager.py:297] Starting prefetching thread
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:35:50 model_runner.py:161] Loading model weights took 6.1114 GB
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:35:50 worker_manager.py:297] Starting prefetching thread
WARNING 05-15 23:35:52 quant_linear_triton.py:557] We should warm up the kernel here 
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8736367d60>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:35:53 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:35:53 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:35:53 quant_linear_triton.py:557] We should warm up the kernel here 
INFO 05-15 23:36:06 ray_gpu_executor.py:275] # GPU blocks: 10679, # CPU blocks: 1310
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f873638d3f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
INFO 05-15 23:36:09 scheduler.py:136] Using DeltaServe policy.
INFO 05-15 23:36:09 block_manager.py:264] disable automatic prefix caching
WARNING 05-15 23:36:09 serving_chat.py:373] No chat template provided. Chat API will not work.
INFO:     Started server process [76277]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     ::1:37652 - "GET /sysinfo HTTP/1.1" 200 OK
Translating from base-model to /vllm/.idea/full_models/Llama-2-13b-hf
Warming up starts
{'id': 21, 'prompt': 'USER: what do you think about the future of iran?\nASSISTANT:', 'timestamp': 0, 'model': 'delta-8', 'min_tokens': 184, 'max_tokens': 184}
Prefetching is disabled
WARNING 05-15 23:36:17 worker_manager.py:344] Missed prefetch, fall back to loading delta 8 now
INFO 05-15 23:36:17 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:36:17 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:36:17 worker_manager.py:344] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:36:17 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:36:17 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:36:17 worker_manager.py:344] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:36:17 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:36:17 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:36:17 worker_manager.py:344] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:36:17 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:36:17 models.py:263] [main] Lossless Compression Disabled
INFO 05-15 23:36:19 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.593 seconds
INFO 05-15 23:36:19 worker_manager.py:364] Main thread loading delta 8 done
INFO 05-15 23:36:19 worker_manager.py:379] [1715787379.16284] activating delta
INFO 05-15 23:36:19 worker_manager.py:384] [1715787379.2594056] CPU -> GPU time: 0.0965
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:36:19 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.077 seconds
INFO 05-15 23:36:19 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:36:19 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.136 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:36:19 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.104 seconds
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:36:19 worker_manager.py:364] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:36:19 worker_manager.py:379] [1715787379.642084] activating delta
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:36:19 worker_manager.py:364] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:36:19 worker_manager.py:379] [1715787379.7143166] activating delta
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:36:19 worker_manager.py:364] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:36:19 worker_manager.py:379] [1715787379.6606703] activating delta
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:36:19 worker_manager.py:384] [1715787379.7750382] CPU -> GPU time: 0.1329
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:36:19 worker_manager.py:384] [1715787379.8085916] CPU -> GPU time: 0.1479
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:36:19 worker_manager.py:384] [1715787379.8647547] CPU -> GPU time: 0.1504
INFO 05-15 23:36:25 metrics.py:276] Avg prompt throughput: 3.9 tokens/s, Avg generation throughput: 3.9 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-15 23:36:30 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-15 23:36:35 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO 05-15 23:36:40 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO 05-15 23:36:45 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO 05-15 23:36:50 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO 05-15 23:36:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO 05-15 23:37:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO:     ::1:37654 - "POST /v1/completions HTTP/1.1" 200 OK
Warming up ends
Issuing queries
sending 1 queries at 0.1
sending 1 queries at 0.30000000000000004
sending 1 queries at 0.4
sending 2 queries at 0.5
sending 1 queries at 0.6000000000000001
sending 1 queries at 0.7000000000000001
sending 1 queries at 1.0
sending 1 queries at 1.3
sending 1 queries at 1.4000000000000001
sending 2 queries at 1.6
sending 1 queries at 1.7000000000000002
sending 4 queries at 2.0
sending 1 queries at 2.2
sending 1 queries at 2.4000000000000004
sending 1 queries at 2.6
sending 1 queries at 3.1
sending 1 queries at 3.2
sending 1 queries at 3.3000000000000003
sending 2 queries at 3.5
sending 2 queries at 3.6
sending 1 queries at 3.9000000000000004
sending 1 queries at 4.0
sending 2 queries at 4.1000000000000005
sending 2 queries at 4.3
sending 2 queries at 4.4
sending 1 queries at 4.6000000000000005
sending 1 queries at 4.7
sending 1 queries at 4.800000000000001
sending 1 queries at 5.1000000000000005
sending 1 queries at 5.2
sending 2 queries at 5.300000000000001
sending 2 queries at 5.5
sending 1 queries at 5.6000000000000005
sending 3 queries at 5.7
sending 2 queries at 5.800000000000001
sending 1 queries at 5.9
sending 1 queries at 6.0
sending 3 queries at 6.5
sending 1 queries at 6.6000000000000005
sending 2 queries at 6.7
sending 4 queries at 6.800000000000001
sending 4 queries at 7.0
sending 2 queries at 7.2
sending 2 queries at 7.5
sending 2 queries at 7.9
sending 1 queries at 8.4
sending 1 queries at 8.5
sending 2 queries at 8.6
sending 4 queries at 8.700000000000001
sending 3 queries at 8.8
sending 1 queries at 9.0
sending 2 queries at 9.1
sending 2 queries at 9.200000000000001
sending 1 queries at 9.3
sending 2 queries at 9.600000000000001
sending 1 queries at 9.700000000000001
sending 1 queries at 9.8
sending 2 queries at 9.9
sending 1 queries at 10.0
sending 2 queries at 10.100000000000001
sending 2 queries at 10.3
sending 2 queries at 10.4
sending 1 queries at 10.600000000000001
sending 1 queries at 10.9
sending 1 queries at 11.0
sending 1 queries at 11.100000000000001
sending 1 queries at 11.200000000000001
sending 2 queries at 11.3
sending 1 queries at 11.600000000000001
sending 1 queries at 11.700000000000001
sending 1 queries at 11.9
sending 1 queries at 12.0
sending 1 queries at 12.100000000000001
sending 2 queries at 12.3
sending 1 queries at 12.5
sending 1 queries at 12.600000000000001
sending 1 queries at 12.9
sending 1 queries at 13.0
sending 2 queries at 13.200000000000001
sending 1 queries at 13.600000000000001
sending 1 queries at 13.700000000000001
sending 1 queries at 13.8
sending 3 queries at 13.9
sending 1 queries at 14.0
sending 1 queries at 14.100000000000001
sending 3 queries at 14.200000000000001
sending 1 queries at 14.3
sending 1 queries at 14.4
sending 1 queries at 14.5
sending 1 queries at 14.600000000000001
sending 1 queries at 14.700000000000001
sending 1 queries at 14.8
sending 1 queries at 14.9
sending 1 queries at 15.100000000000001
sending 2 queries at 15.200000000000001
sending 1 queries at 15.5
sending 1 queries at 15.700000000000001
sending 2 queries at 15.8
sending 1 queries at 16.1
sending 1 queries at 16.2
sending 2 queries at 17.0
sending 2 queries at 17.2
sending 2 queries at 17.400000000000002
sending 1 queries at 17.6
sending 1 queries at 17.8
sending 3 queries at 17.900000000000002
sending 1 queries at 18.0
sending 1 queries at 18.1
sending 1 queries at 18.3
sending 1 queries at 18.5
sending 1 queries at 18.900000000000002
sending 2 queries at 19.1
sending 1 queries at 19.200000000000003
sending 2 queries at 19.3
sending 4 queries at 19.400000000000002
sending 2 queries at 19.6
sending 1 queries at 19.700000000000003
sending 1 queries at 20.0
sending 3 queries at 20.1
sending 2 queries at 20.3
sending 1 queries at 20.400000000000002
sending 1 queries at 20.700000000000003
sending 1 queries at 20.8
sending 1 queries at 20.900000000000002
sending 1 queries at 21.0
sending 1 queries at 21.1
sending 3 queries at 21.200000000000003
sending 1 queries at 21.3
sending 1 queries at 21.6
sending 1 queries at 21.700000000000003
sending 2 queries at 21.8
sending 3 queries at 21.900000000000002
sending 1 queries at 22.0
sending 1 queries at 22.1
sending 4 queries at 22.200000000000003
sending 1 queries at 22.3
sending 1 queries at 22.400000000000002
sending 1 queries at 22.5
sending 1 queries at 22.700000000000003
sending 1 queries at 23.200000000000003
sending 1 queries at 23.3
sending 3 queries at 23.400000000000002
sending 2 queries at 23.6
sending 1 queries at 23.700000000000003
sending 1 queries at 23.8
sending 1 queries at 24.0
sending 1 queries at 24.1
sending 1 queries at 24.3
sending 1 queries at 24.400000000000002
sending 1 queries at 24.5
sending 1 queries at 24.6
sending 1 queries at 24.900000000000002
sending 1 queries at 25.0
sending 1 queries at 25.1
sending 1 queries at 25.400000000000002
sending 1 queries at 25.5
sending 2 queries at 25.6
sending 3 queries at 25.700000000000003
sending 2 queries at 25.8
sending 2 queries at 25.900000000000002
sending 1 queries at 26.0
sending 2 queries at 26.200000000000003
sending 2 queries at 26.3
sending 2 queries at 26.5
sending 1 queries at 26.6
sending 3 queries at 26.700000000000003
sending 1 queries at 26.8
sending 1 queries at 27.1
sending 1 queries at 27.3
sending 1 queries at 27.400000000000002
sending 2 queries at 27.700000000000003
sending 1 queries at 27.8
sending 1 queries at 27.900000000000002
sending 1 queries at 28.200000000000003
sending 4 queries at 28.3
sending 1 queries at 28.6
sending 1 queries at 28.700000000000003
sending 1 queries at 28.900000000000002
sending 2 queries at 29.0
sending 2 queries at 29.1
sending 1 queries at 29.6
sending 2 queries at 29.8
Prefetching is disabled
WARNING 05-15 23:37:05 worker_manager.py:344] Missed prefetch, fall back to loading delta 9 now
INFO 05-15 23:37:05 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:05 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:05 worker_manager.py:344] Missed prefetch, fall back to loading delta 9 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:05 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:05 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:05 worker_manager.py:344] Missed prefetch, fall back to loading delta 9 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:05 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:05 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:05 worker_manager.py:344] Missed prefetch, fall back to loading delta 9 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:05 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:05 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:06 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.631 seconds
INFO 05-15 23:37:07 worker_manager.py:364] Main thread loading delta 9 done
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:07 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.017 seconds
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:07 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.089 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:07 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.053 seconds
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:07 worker_manager.py:364] Main thread loading delta 9 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:07 worker_manager.py:364] Main thread loading delta 9 done
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:07 worker_manager.py:364] Main thread loading delta 9 done
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:08 metrics.py:276] Avg prompt throughput: 3.0 tokens/s, Avg generation throughput: 2.6 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
WARNING 05-15 23:37:08 worker_manager.py:344] Missed prefetch, fall back to loading delta 1 now
INFO 05-15 23:37:08 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:08 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:08 worker_manager.py:344] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:08 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:08 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:08 worker_manager.py:344] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:08 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:08 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:08 worker_manager.py:344] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:08 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:08 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:09 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.580 seconds
INFO 05-15 23:37:09 worker_manager.py:364] Main thread loading delta 1 done
Prefetching is disabled
Prefetching is disabled
WARNING 05-15 23:37:09 worker_manager.py:344] Missed prefetch, fall back to loading delta 3 now
INFO 05-15 23:37:09 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:09 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:10 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.060 seconds
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:10 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.078 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:10 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.084 seconds
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:10 worker_manager.py:364] Main thread loading delta 1 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:10 worker_manager.py:364] Main thread loading delta 1 done
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:10 worker_manager.py:364] Main thread loading delta 1 done
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:10 worker_manager.py:344] Missed prefetch, fall back to loading delta 3 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:10 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:10 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:10 worker_manager.py:344] Missed prefetch, fall back to loading delta 3 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:10 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:10 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:10 worker_manager.py:344] Missed prefetch, fall back to loading delta 3 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:10 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:10 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:11 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.605 seconds
INFO 05-15 23:37:11 worker_manager.py:364] Main thread loading delta 3 done
WARNING 05-15 23:37:11 worker_manager.py:344] Missed prefetch, fall back to loading delta 5 now
INFO 05-15 23:37:11 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:11 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:12 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.064 seconds
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:12 worker_manager.py:364] Main thread loading delta 3 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:12 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.054 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:12 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.051 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:12 worker_manager.py:364] Main thread loading delta 3 done
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:12 worker_manager.py:344] Missed prefetch, fall back to loading delta 5 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:12 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:12 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:12 worker_manager.py:364] Main thread loading delta 3 done
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:12 worker_manager.py:344] Missed prefetch, fall back to loading delta 5 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:12 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:12 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:12 worker_manager.py:344] Missed prefetch, fall back to loading delta 5 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:12 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:12 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:13 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.569 seconds
INFO 05-15 23:37:13 worker_manager.py:364] Main thread loading delta 5 done
Prefetching is disabled
Prefetching is disabled
WARNING 05-15 23:37:13 worker_manager.py:344] Missed prefetch, fall back to loading delta 6 now
INFO 05-15 23:37:13 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:13 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:14 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.097 seconds
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:14 worker_manager.py:364] Main thread loading delta 5 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:14 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.095 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:14 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.063 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:14 worker_manager.py:364] Main thread loading delta 5 done
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:14 worker_manager.py:344] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:14 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:14 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:14 worker_manager.py:364] Main thread loading delta 5 done
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:14 worker_manager.py:344] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:14 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:14 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:14 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.578 seconds
INFO 05-15 23:37:14 worker_manager.py:364] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:14 worker_manager.py:344] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:14 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:14 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
WARNING 05-15 23:37:15 worker_manager.py:344] Missed prefetch, fall back to loading delta 7 now
INFO 05-15 23:37:15 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:15 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:16 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.549 seconds
INFO 05-15 23:37:16 worker_manager.py:364] Main thread loading delta 7 done
WARNING 05-15 23:37:16 worker_manager.py:344] Missed prefetch, fall back to loading delta 11 now
INFO 05-15 23:37:16 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:16 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:16 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.048 seconds
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:16 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.115 seconds
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:17 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.073 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:16 worker_manager.py:364] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:17 worker_manager.py:364] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:17 worker_manager.py:364] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:17 worker_manager.py:344] Missed prefetch, fall back to loading delta 7 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:17 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:17 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:17 worker_manager.py:344] Missed prefetch, fall back to loading delta 7 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:17 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:17 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:17 worker_manager.py:344] Missed prefetch, fall back to loading delta 7 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:17 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:17 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:18 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.560 seconds
INFO 05-15 23:37:18 worker_manager.py:364] Main thread loading delta 11 done
Prefetching is disabled
Prefetching is disabled
WARNING 05-15 23:37:18 worker_manager.py:344] Missed prefetch, fall back to loading delta 12 now
INFO 05-15 23:37:18 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:18 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:19 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.995 seconds
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:19 worker_manager.py:364] Main thread loading delta 7 done
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:19 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.060 seconds
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:19 worker_manager.py:364] Main thread loading delta 7 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:19 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.098 seconds
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:19 worker_manager.py:344] Missed prefetch, fall back to loading delta 11 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:19 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:19 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:19 worker_manager.py:364] Main thread loading delta 7 done
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:19 worker_manager.py:344] Missed prefetch, fall back to loading delta 11 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:19 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:19 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:19 worker_manager.py:344] Missed prefetch, fall back to loading delta 11 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:19 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:19 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:19 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 20 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.4%, CPU KV cache usage: 0.0%
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:20 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.531 seconds
INFO 05-15 23:37:20 worker_manager.py:364] Main thread loading delta 12 done
Prefetching is disabled
WARNING 05-15 23:37:20 worker_manager.py:344] Missed prefetch, fall back to loading delta 14 now
INFO 05-15 23:37:20 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:20 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:21 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.004 seconds
Prefetching is disabled
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:21 worker_manager.py:364] Main thread loading delta 11 done
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:21 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.046 seconds
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:21 worker_manager.py:344] Missed prefetch, fall back to loading delta 12 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:21 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:21 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:21 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.053 seconds
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:21 worker_manager.py:364] Main thread loading delta 11 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:21 worker_manager.py:364] Main thread loading delta 11 done
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:21 worker_manager.py:344] Missed prefetch, fall back to loading delta 12 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:21 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:21 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:21 worker_manager.py:344] Missed prefetch, fall back to loading delta 12 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:21 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:21 models.py:263] [main] Lossless Compression Disabled
INFO 05-15 23:37:21 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.534 seconds
INFO 05-15 23:37:22 worker_manager.py:364] Main thread loading delta 14 done
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:23 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.955 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:23 worker_manager.py:364] Main thread loading delta 12 done
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:23 worker_manager.py:344] Missed prefetch, fall back to loading delta 14 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:23 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:23 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:23 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.952 seconds
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:23 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.951 seconds
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:23 worker_manager.py:364] Main thread loading delta 12 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:23 worker_manager.py:364] Main thread loading delta 12 done
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:23 worker_manager.py:344] Missed prefetch, fall back to loading delta 14 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:23 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:23 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:23 worker_manager.py:344] Missed prefetch, fall back to loading delta 14 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:23 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:23 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:25 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.950 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:25 worker_manager.py:364] Main thread loading delta 14 done
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:25 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.006 seconds
Prefetching is disabled
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:25 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.943 seconds
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:26 worker_manager.py:364] Main thread loading delta 14 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:26 worker_manager.py:364] Main thread loading delta 14 done
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:28 metrics.py:276] Avg prompt throughput: 55.3 tokens/s, Avg generation throughput: 2.1 tokens/s, Running: 19 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.4%, CPU KV cache usage: 0.0%
INFO:     ::1:37804 - "POST /v1/completions HTTP/1.1" 200 OK
WARNING 05-15 23:37:28 worker_manager.py:344] Missed prefetch, fall back to loading delta 2 now
INFO 05-15 23:37:28 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:28 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:28 worker_manager.py:344] Missed prefetch, fall back to loading delta 2 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:28 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:28 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:28 worker_manager.py:344] Missed prefetch, fall back to loading delta 2 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:28 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:28 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:28 worker_manager.py:344] Missed prefetch, fall back to loading delta 2 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:28 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:28 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:30 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.585 seconds
INFO 05-15 23:37:30 worker_manager.py:364] Main thread loading delta 2 done
Prefetching is disabled
WARNING 05-15 23:37:30 worker_manager.py:344] Missed prefetch, fall back to loading delta 4 now
Prefetching is disabled
INFO 05-15 23:37:30 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:30 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:31 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.084 seconds
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:31 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.078 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:31 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.065 seconds
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:31 worker_manager.py:364] Main thread loading delta 2 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:31 worker_manager.py:364] Main thread loading delta 2 done
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:31 worker_manager.py:364] Main thread loading delta 2 done
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:31 worker_manager.py:344] Missed prefetch, fall back to loading delta 4 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:31 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:31 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:31 worker_manager.py:344] Missed prefetch, fall back to loading delta 4 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:31 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:31 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:31 worker_manager.py:344] Missed prefetch, fall back to loading delta 4 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:31 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:31 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:32 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.558 seconds
INFO 05-15 23:37:32 worker_manager.py:364] Main thread loading delta 4 done
Prefetching is disabled
WARNING 05-15 23:37:32 worker_manager.py:344] Missed prefetch, fall back to loading delta 10 now
INFO 05-15 23:37:32 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:32 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:33 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.038 seconds
Prefetching is disabled
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:33 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.102 seconds
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:33 worker_manager.py:364] Main thread loading delta 4 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:33 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.085 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:33 worker_manager.py:364] Main thread loading delta 4 done
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:33 worker_manager.py:364] Main thread loading delta 4 done
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:33 worker_manager.py:344] Missed prefetch, fall back to loading delta 10 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:33 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:33 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:33 worker_manager.py:344] Missed prefetch, fall back to loading delta 10 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:33 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:33 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:33 worker_manager.py:344] Missed prefetch, fall back to loading delta 10 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:33 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:33 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-15 23:37:34 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.524 seconds
INFO 05-15 23:37:34 worker_manager.py:364] Main thread loading delta 10 done
Prefetching is disabled
WARNING 05-15 23:37:34 worker_manager.py:344] Missed prefetch, fall back to loading delta 13 now
INFO 05-15 23:37:34 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:34 models.py:263] [main] Lossless Compression Disabled
total threads: 273
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:35 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.966 seconds
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:35 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.046 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:35 worker_manager.py:364] Main thread loading delta 10 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:35 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.043 seconds
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:35 worker_manager.py:344] Missed prefetch, fall back to loading delta 13 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:35 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:35 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:35 worker_manager.py:364] Main thread loading delta 10 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:35 worker_manager.py:364] Main thread loading delta 10 done
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:35 worker_manager.py:344] Missed prefetch, fall back to loading delta 13 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:35 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:35 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:35 worker_manager.py:344] Missed prefetch, fall back to loading delta 13 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:35 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:35 models.py:263] [main] Lossless Compression Disabled
INFO 05-15 23:37:36 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.879 seconds
INFO 05-15 23:37:36 worker_manager.py:364] Main thread loading delta 13 done
WARNING 05-15 23:37:36 worker_manager.py:344] Missed prefetch, fall back to loading delta 15 now
INFO 05-15 23:37:36 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-15 23:37:36 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:37 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.040 seconds
INFO 05-15 23:37:37 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.545 seconds
INFO 05-15 23:37:38 worker_manager.py:364] Main thread loading delta 15 done
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:37 worker_manager.py:364] Main thread loading delta 13 done
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:38 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.083 seconds
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:38 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.064 seconds
[36m(RayWorkerVllm pid=84761)[0m WARNING 05-15 23:37:38 worker_manager.py:344] Missed prefetch, fall back to loading delta 15 now
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:38 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:38 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:38 worker_manager.py:364] Main thread loading delta 13 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:38 worker_manager.py:364] Main thread loading delta 13 done
[36m(RayWorkerVllm pid=84425)[0m WARNING 05-15 23:37:38 worker_manager.py:344] Missed prefetch, fall back to loading delta 15 now
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:38 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:38 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=84609)[0m WARNING 05-15 23:37:38 worker_manager.py:344] Missed prefetch, fall back to loading delta 15 now
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:38 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:38 models.py:263] [main] Lossless Compression Disabled
INFO 05-15 23:37:39 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 125 reqs, Swapped: 0 reqs, Pending: 92 reqs, GPU KV cache usage: 3.3%, CPU KV cache usage: 0.0%
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:40 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.919 seconds
[36m(RayWorkerVllm pid=84761)[0m INFO 05-15 23:37:41 worker_manager.py:364] Main thread loading delta 15 done
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:41 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.983 seconds
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:41 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.987 seconds
[36m(RayWorkerVllm pid=84425)[0m INFO 05-15 23:37:41 worker_manager.py:364] Main thread loading delta 15 done
[36m(RayWorkerVllm pid=84609)[0m INFO 05-15 23:37:41 worker_manager.py:364] Main thread loading delta 15 done
INFO 05-15 23:37:49 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 125 reqs, Swapped: 0 reqs, Pending: 92 reqs, GPU KV cache usage: 3.3%, CPU KV cache usage: 0.0%
INFO 05-15 23:37:59 metrics.py:276] Avg prompt throughput: 409.3 tokens/s, Avg generation throughput: 10.6 tokens/s, Running: 192 reqs, Swapped: 0 reqs, Pending: 80 reqs, GPU KV cache usage: 5.8%, CPU KV cache usage: 0.0%
INFO 05-15 23:38:05 metrics.py:276] Avg prompt throughput: 689.3 tokens/s, Avg generation throughput: 12.4 tokens/s, Running: 192 reqs, Swapped: 0 reqs, Pending: 80 reqs, GPU KV cache usage: 5.8%, CPU KV cache usage: 0.0%
INFO 05-15 23:38:15 metrics.py:276] Avg prompt throughput: 316.8 tokens/s, Avg generation throughput: 6.4 tokens/s, Running: 256 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 7.9%, CPU KV cache usage: 0.0%
INFO:     ::1:38106 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37838 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:38:20 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 93.0 tokens/s, Running: 253 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 8.2%, CPU KV cache usage: 0.0%
INFO:     ::1:38312 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38128 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:38:26 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 91.9 tokens/s, Running: 249 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 8.4%, CPU KV cache usage: 0.0%
INFO:     ::1:37834 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37798 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38200 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38388 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38142 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37846 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37852 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37964 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37848 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:38:31 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 89.9 tokens/s, Running: 241 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 8.4%, CPU KV cache usage: 0.0%
INFO:     ::1:37842 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38258 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38372 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:38:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 87.1 tokens/s, Running: 233 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 8.2%, CPU KV cache usage: 0.0%
INFO:     ::1:37940 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37938 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38122 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37862 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38322 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38361 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38170 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38078 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38328 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38194 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38260 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:38:42 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 84.5 tokens/s, Running: 226 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 8.3%, CPU KV cache usage: 0.0%
INFO:     ::1:37854 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38178 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38000 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37795 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:38:48 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 82.7 tokens/s, Running: 223 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 8.5%, CPU KV cache usage: 0.0%
INFO:     ::1:38028 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38316 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38120 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38080 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38364 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:38:53 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 79.9 tokens/s, Running: 220 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 8.8%, CPU KV cache usage: 0.0%
INFO:     ::1:38374 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:38:59 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 79.6 tokens/s, Running: 218 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 9.1%, CPU KV cache usage: 0.0%
INFO:     ::1:38338 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38308 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38366 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:39:04 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 78.5 tokens/s, Running: 216 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 9.4%, CPU KV cache usage: 0.0%
INFO 05-15 23:39:09 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 41.6 tokens/s, Running: 216 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 9.6%, CPU KV cache usage: 0.0%
INFO:     ::1:38210 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38104 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:39:16 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 105.3 tokens/s, Running: 212 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 9.6%, CPU KV cache usage: 0.0%
INFO:     ::1:38034 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37972 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37876 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:39:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 76.4 tokens/s, Running: 210 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 9.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38068 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:39:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 76.3 tokens/s, Running: 210 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 9.9%, CPU KV cache usage: 0.0%
INFO 05-15 23:39:32 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 76.3 tokens/s, Running: 209 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 10.2%, CPU KV cache usage: 0.0%
INFO:     ::1:37864 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:39:38 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 76.3 tokens/s, Running: 208 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 10.4%, CPU KV cache usage: 0.0%
INFO:     ::1:37936 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37826 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:39:43 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 75.7 tokens/s, Running: 206 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 10.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38304 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:39:49 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 74.7 tokens/s, Running: 204 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 10.9%, CPU KV cache usage: 0.0%
INFO:     ::1:37806 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37962 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38270 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38036 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:39:54 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 74.3 tokens/s, Running: 202 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 11.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38114 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38324 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:39:59 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 37.4 tokens/s, Running: 200 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 11.1%, CPU KV cache usage: 0.0%
INFO:     ::1:38186 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:40:05 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 108.0 tokens/s, Running: 198 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 11.1%, CPU KV cache usage: 0.0%
INFO:     ::1:37884 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:40:10 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 72.5 tokens/s, Running: 197 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 11.3%, CPU KV cache usage: 0.0%
INFO:     ::1:37946 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37792 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:40:16 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 72.6 tokens/s, Running: 196 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 11.6%, CPU KV cache usage: 0.0%
INFO:     ::1:37822 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:40:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 72.0 tokens/s, Running: 194 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 11.8%, CPU KV cache usage: 0.0%
INFO:     ::1:37904 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:40:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 71.4 tokens/s, Running: 193 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 12.0%, CPU KV cache usage: 0.0%
INFO:     ::1:37898 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37966 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:40:32 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 70.6 tokens/s, Running: 191 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 12.2%, CPU KV cache usage: 0.0%
INFO:     ::1:38292 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:40:38 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 70.6 tokens/s, Running: 191 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 12.3%, CPU KV cache usage: 0.0%
INFO:     ::1:38092 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:40:43 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 70.7 tokens/s, Running: 190 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 12.4%, CPU KV cache usage: 0.0%
INFO 05-15 23:40:48 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 70.6 tokens/s, Running: 189 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 12.5%, CPU KV cache usage: 0.0%
INFO:     ::1:38272 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:40:54 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 66.9 tokens/s, Running: 189 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 12.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38282 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:40:59 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 70.0 tokens/s, Running: 185 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 12.8%, CPU KV cache usage: 0.0%
INFO:     ::1:37778 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38302 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37920 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:41:05 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 68.7 tokens/s, Running: 184 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38248 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38310 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:41:10 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 68.4 tokens/s, Running: 182 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.2%, CPU KV cache usage: 0.0%
INFO:     ::1:38038 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37900 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37866 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37994 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:41:15 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 66.7 tokens/s, Running: 176 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 12.9%, CPU KV cache usage: 0.0%
INFO:     ::1:38286 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37782 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38010 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38334 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:41:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 66.1 tokens/s, Running: 174 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 12.9%, CPU KV cache usage: 0.0%
INFO:     ::1:38268 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:41:26 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 65.7 tokens/s, Running: 174 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.1%, CPU KV cache usage: 0.0%
INFO:     ::1:37880 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:41:31 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 65.4 tokens/s, Running: 172 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.1%, CPU KV cache usage: 0.0%
INFO:     ::1:38064 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38246 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:41:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 64.6 tokens/s, Running: 171 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.2%, CPU KV cache usage: 0.0%
INFO:     ::1:38066 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:41:42 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 64.4 tokens/s, Running: 170 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.4%, CPU KV cache usage: 0.0%
INFO 05-15 23:41:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 64.5 tokens/s, Running: 169 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.6%, CPU KV cache usage: 0.0%
INFO:     ::1:37950 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:41:53 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 64.0 tokens/s, Running: 168 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.8%, CPU KV cache usage: 0.0%
INFO:     ::1:37974 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38266 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:41:58 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 63.4 tokens/s, Running: 167 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 14.0%, CPU KV cache usage: 0.0%
INFO:     ::1:37914 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38118 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37800 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:42:03 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 62.3 tokens/s, Running: 163 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.8%, CPU KV cache usage: 0.0%
INFO:     ::1:38188 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:42:08 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 62.2 tokens/s, Running: 160 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.6%, CPU KV cache usage: 0.0%
INFO:     ::1:38234 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38046 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38380 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37983 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38296 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:42:14 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 61.2 tokens/s, Running: 157 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.5%, CPU KV cache usage: 0.0%
INFO:     ::1:38098 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:42:19 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 59.8 tokens/s, Running: 155 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.4%, CPU KV cache usage: 0.0%
INFO:     ::1:38288 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38048 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:42:24 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 59.9 tokens/s, Running: 155 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38042 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38014 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:42:29 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 59.5 tokens/s, Running: 152 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38052 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37817 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:42:34 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 58.8 tokens/s, Running: 151 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.9%, CPU KV cache usage: 0.0%
INFO:     ::1:38274 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38132 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:42:39 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 29.9 tokens/s, Running: 149 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38044 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:42:45 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 84.8 tokens/s, Running: 146 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.6%, CPU KV cache usage: 0.0%
INFO:     ::1:37812 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37888 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37856 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38008 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:42:50 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 53.9 tokens/s, Running: 142 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.4%, CPU KV cache usage: 0.0%
INFO:     ::1:38168 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37784 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38130 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:42:55 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 54.5 tokens/s, Running: 140 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.3%, CPU KV cache usage: 0.0%
INFO:     ::1:37802 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38300 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:43:00 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 54.1 tokens/s, Running: 137 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.3%, CPU KV cache usage: 0.0%
INFO:     ::1:38278 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38226 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38318 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38192 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38020 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38222 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:43:05 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 52.9 tokens/s, Running: 132 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.1%, CPU KV cache usage: 0.0%
INFO:     ::1:38347 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:43:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 51.4 tokens/s, Running: 131 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.2%, CPU KV cache usage: 0.0%
INFO:     ::1:38182 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:43:16 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 51.3 tokens/s, Running: 130 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.3%, CPU KV cache usage: 0.0%
INFO:     ::1:38359 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:43:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 51.1 tokens/s, Running: 129 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.4%, CPU KV cache usage: 0.0%
INFO:     ::1:37796 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:43:26 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 50.4 tokens/s, Running: 128 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.4%, CPU KV cache usage: 0.0%
INFO:     ::1:38208 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:43:31 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 50.6 tokens/s, Running: 126 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.3%, CPU KV cache usage: 0.0%
INFO:     ::1:38074 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38197 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:43:36 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 49.8 tokens/s, Running: 126 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.4%, CPU KV cache usage: 0.0%
INFO 05-15 23:43:41 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 50.1 tokens/s, Running: 125 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.3%, CPU KV cache usage: 0.0%
INFO:     ::1:38050 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:43:46 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 49.5 tokens/s, Running: 125 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.5%, CPU KV cache usage: 0.0%
INFO:     ::1:37910 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:43:51 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 49.5 tokens/s, Running: 123 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.5%, CPU KV cache usage: 0.0%
INFO:     ::1:37874 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:43:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 48.7 tokens/s, Running: 122 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.7%, CPU KV cache usage: 0.0%
INFO:     ::1:37988 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:44:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 47.8 tokens/s, Running: 121 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38306 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:44:06 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 48.0 tokens/s, Running: 119 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.3%, CPU KV cache usage: 0.0%
INFO:     ::1:38236 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38158 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38004 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:44:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 47.3 tokens/s, Running: 118 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.3%, CPU KV cache usage: 0.0%
INFO 05-15 23:44:19 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 47.5 tokens/s, Running: 118 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.5%, CPU KV cache usage: 0.0%
INFO:     ::1:38368 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:37773 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38394 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:44:26 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 46.9 tokens/s, Running: 115 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.4%, CPU KV cache usage: 0.0%
INFO:     ::1:38002 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38072 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:44:34 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 45.8 tokens/s, Running: 111 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.3%, CPU KV cache usage: 0.0%
INFO:     ::1:38102 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38022 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:44:39 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 39.3 tokens/s, Running: 111 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.5%, CPU KV cache usage: 0.0%
INFO:     ::1:38156 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38154 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:44:46 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 49.2 tokens/s, Running: 109 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.4%, CPU KV cache usage: 0.0%
INFO:     ::1:37836 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:44:54 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 43.7 tokens/s, Running: 108 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.4%, CPU KV cache usage: 0.0%
INFO:     ::1:38340 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38220 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38180 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:44:59 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 36.8 tokens/s, Running: 105 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.2%, CPU KV cache usage: 0.0%
INFO:     ::1:37882 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-15 23:45:06 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 48.4 tokens/s, Running: 103 reqs, Swapped: 0 reqs, Pending: 16 reqs, GPU KV cache usage: 13.2%, CPU KV cache usage: 0.0%
INFO:     ::1:37830 - "POST /v1/completions HTTP/1.1" 200 OK
